pkg/report/report.go

```diff
@@ -55,12 +55,29 @@ type report struct {
 	sps *secondPoints
 }
 
+// Stats exposes results raw data.
+type Stats struct {
```

Comment:
Figured I only need raw data, so that I can store in CSV. If we do, seems like we will have to duplicate a lot of `*report` methods? (e.g. histogram)